privaterelay/settings.py

```diff
@@ -128,10 +128,14 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.sites',
+
+    'django_gulp',
```

Comment:
Would love to figure out if we could use `django_compressor` and `django-libsass` for protocol instead? (https://www.accordbox.com/blog/how-use-scss-sass-your-django-project-python-way/)

Reply:
FWIW – I have this deploying correctly to Heroku/Circle CI in this PR (https://github.com/mozilla/fx-private-relay/pull/859) 

While we aren't using Protocol directly right now (just Gulp/SCSS) – We're using it via NPM (it's a [NPM package](https://www.npmjs.com/package/@mozilla-protocol/core)). Updates for it in the future will be easier this way AND if we want to concat/use their JS libraries, and other assets (fonts, images, etc.) – gulp will be needed to copy/manipulate assets. 

I recommend stickin' it out if we can.